docker/adelphi/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM python:3.9-alpine3.12
+
+RUN apk add build-base libffi-dev python3-dev libev-dev
```

Comment:
Need to check on exactly what's in these packages but my suspicion would be we could subsequently remove some/all of them after compilation in order to cut down on image size.